src/gossip/active_set.zig

```diff
@@ -177,12 +177,13 @@ test "init/denit" {
 test "gracefully rotates with duplicate contact ids" {
     const alloc = std.testing.allocator;
 
-    var rng = std.rand.DefaultPrng.init(100);
+    // NOTE: DefaultPrng == Xoshiro256, so mb it would be better to use only DefaultPrng
```

Comment:
What are these comments getting at? "it would be better to use only DefaultPrng" seems like a TODO comment to me?

Reply:
"it would be better to use only DefaultPrng" sounds like "Remove part which uses Xoshiro256 directly" and may lead to removing important tests

Reply:
I don't understand what you mean. Why would we be removing tests? 
I might be missing something really obvious I'm sorry.